src/main/kotlin/gg/ingot/iron/sql/controller/Controller.kt

```diff
@@ -68,6 +100,11 @@ sealed interface Controller {
      * @since 1.0
      */
     fun execute(@Language("SQL") statement: String): Boolean
+
+    private companion object {
+        /** The regex for SQL placeholders. */
+        val SQL_PLACEHOLDER_REGEX = """:(\w+)""".toRegex()
```

Comment:
Do not use regex for variable handling, either parse the sql or properly handle cases such as the following:
`WHERE abc = "hello :world"` where `:world` might also be a parameter you're passing in

Reply:
Is this even allowed in normal SQL?

Reply:
Yes, it gets treated as regular text, but might not in how you have your replacement setup

Reply:
Fixed the regex rule